src/editing_tools/AdjustTool.vala

```diff
@@ -342,13 +342,16 @@ public class EditingTools.AdjustTool : EditingTool {
                 // multiple successive enhances are as good as a single, as long as it's on the
                 // same photo
                 return photo.equals (owner.canvas.photo);
+            } else {
+                return false;
```

Comment:
This makes the following code unreachable.  I do not think this change should be made.

Reply:
Yeah I agree. Lemme change that